.github/workflows/conda.yml

```diff
@@ -49,6 +49,11 @@ jobs:
           which python
           pip list
           mamba list
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v1
+        with:
+          version: '3.x'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We should probably remove this and instead add `libprotobuf=3` to the nightly recipe's build requirements (in a similar fashion to [what's being done in ADP](https://github.com/apache/arrow-datafusion-python/blob/af1a8ebade3d537ce24f6f01a037a56177a917fa/conda/recipes/meta.yaml#L40)) so that compilers are handled entirely through conda-forge once this gets pushed into a stable release

```suggestion
```